Add "Upgrade to Airflow 2" docs to Astro (#844)
astronomer/docs: astro/data-plane-activation.md

> When you meet with one of our engineers to activate your data plane, **expect it to take about an hour**. By the end of the session, you should have your **first pipeline deployed in your own Astro environment**!

Reviewer: ```suggestion
It takes approximately an hour to activate your data plane, You'll work with an Astronomer engineer and you'll have your first pipeline deployed in your Astro environment by the end of the session.
```

> ### What to Bring and Know

Reviewer: ```suggestion
### Requirements
```

> Your data plane will be deployed into a clean, dedicated AWS account or GCP project. With our fully managed service, we’ll take complete responsibility for the operations of this account, as described in our [Shared Responsibility Model](shared-responsibility-model.md).

Reviewer: ```suggestion
Astronomer deploys your data plane to a dedicated Amazon Web Services (AWS) account or a Google Cloud project (GCP). Astronomer is responsible for the operations of this account. See [Shared Responsibility Model](shared-responsibility-model.md).
```

This model allows us to get you started quickly, providing cloud-grade reliability and seamless connection to all of your data services. If you decide not to proceed with Astro, this account can be deleted in its entirety.

By default, the Astronomer account has no access to your data services. We’ll guide you through how to make these connections securely, whether peering VPCs or making direct connections.

> To test an existing Airflow project on Astro, your Airflow code needs to be compatible with Airflow 2. For a complete guide on how to migrate an open source Airflow project running 1.10.15 or lower to Astro, see [Upgrade to Airflow 2.0+](upgrade-to-airflow-2.md).

Reviewer: ```suggestion
To test an existing Airflow project on Astro, your Airflow code needs to be compatible with Airflow 2. For information about migrating an open source Airflow project running 1.10.15 or lower to Astro, see [Upgrade to Airflow 2.0+](upgrade-to-airflow-2.md).
```
